### DISCUSSION TITLE
cl_intel_unified_shared_memory says nothing about memory model

### DISCUSSION THREAD

**Discussion by b-sumner** [intel/llvm#6716] [General]
I realize that this is WIP, but the lack of discussion about the interaction with the memory model in the current draft seems to be a substantial gap that really needs to be filled.

**Comment by romanovvlad** [comment]
Tagging @jbrodman @gmlueck @bashbaug

**Reply by bashbaug** [reply]
I have an internal draft that describes many of the interactions with the memory model.  It's a little old, and as I recall it's still slightly incomplete in places, but I can create a WIP pull request for additional discussion if that would be useful.
